### ISSUE TITLE
Problems with OpenMP detection

### ISSUE THREAD

**Issue by jbmouret** [resibots/limbo#12]
- it does not work with clang
`err: clang: error: unknown argument: '-qsmp=omp'`

- it should be optional! We do not need openMP

```
./waf configure     
Setting top to                           : /private/tmp/limbo 
Setting out to                           : /private/tmp/limbo/build 
Checking for 'g++' (c++ compiler)        : /usr/bin/g++ 
Checking for 'gcc' (c compiler)          : /usr/bin/gcc 
Checking boost includes                  : 1_58 
Checking boost libs                      : ok 
Checking for compiler option to support OpenMP : Not supported 
OpenMP is not supported
(complete log in /private/tmp/limbo/build/config.log)
```

**Comment by fedeallocati** [comment]
Could you attach the config.log?

**Comment by fedeallocati** [comment]
Nevermind, just fixed it, will make branch and PR

**Comment by jbmouret** [comment]
[config.txt](https://github.com/resibots/limbo/files/15022/config.txt)

**Comment by costashatz** [comment]
Should be fixed with 3e9ab6d

**Comment by fedeallocati** [comment]
Fixed
